src/overmap.cpp

```diff
@@ -1797,7 +1797,8 @@ tripoint overmap::draw_overmap(const tripoint &orig, bool debug_mongroup, const
     std::string action;
     do {
         timeout( BLINK_SPEED );
-        bool show_explored = uistate.overmap_blinking || uistate.overmap_show_overlays;
+        bool show_explored = (uistate.overmap_blinking || uistate.overmap_show_overlays) ||
```

Comment:
What you changed this to is logically equivalent: http://www.wolframalpha.com/input/?i=%28P+||+Q%29+||+%28!P+%26%26+Q%29